### ISSUE TITLE
Backside/frontside glitching in Safari

### ISSUE THREAD

**Issue by canpoyrazoglu** [simeydotme/pokemon-cards-css#10]
The effects look great!

However, on Safari 16.1 on Mac, it glitches and alternates rapidly between backside and frontside as I move the cursor:

https://user-images.githubusercontent.com/4348413/198997278-5e928e62-e594-4452-9670-97e6767462ac.mov

It occurs also when the card is expanded by tapping, so it's not isolated to thumbnails only.

On the same machine, Chrome and Firefox render perfectly.

**Comment by simeydotme** [comment]
Thanks, @canpoyrazoglu  

It's something I've known about since it was a bit late to try and mitigate it ... I will get around to refactoring the code at some point and hope I can resolve the layering/gfx issues with Safari by changing the way the card's backside is rendered.

I'll leave this open as inspiration!

**Comment by james-work-account** [comment]
https://stackoverflow.com/questions/36451688/css-card-flip-in-safari/36451979#36451979 I'm not on an Apple device to test, but does adding -webkit prefixes fix this?

**Comment by canpoyrazoglu** [comment]
It's (almost totally) fixed with the latest PR. See [My comment](https://github.com/simeydotme/pokemon-cards-css/pull/11#issuecomment-1297917452) with video demoing a small yet not super-important issue.

**Comment by simeydotme** [comment]
I just pushed another commit where I've basically fixed all the Safari bugs except one thing that is a pure engine-limit.
https://github.com/simeydotme/pokemon-cards-css/commit/2ff2a5a9c17424c21842af9d881011c76cda9a5e

---

Safari was, for some reason, not handling image sizes the same as FF/Chrome when only a single value was provided. The resolution was to explicitly set the x & y for all background image sizes.

I think I've also improved the performance on Safari a little by setting the transform: translateZ on each layer, it seems Safari can optimise the rendering better... or soemthing?
